tests/cut-plane/test_detection.py

```diff
@@ -56,64 +58,76 @@ def test_success_function():
     bin_width = 10
     params = rot_x, rot_y, rot_z, transl_x, transl_y, transl_z
     res = _success_function(params, _get_points(), bin_width=bin_width)
-    assert_equal(res, 513-1830)
+    assert res == 513-1830
 
 
+@pytest.mark.skipif(sys.version_info >= (3, 7), reason="requires python3.6")
 def test_minimize():
```

Comment:
this test has been split for py36 and other higher python versions as `scipy.optimize.minimize` generates slightly different results for them.

Reply:
so shouldn't this test depends on the version of scipy instead of the version of Python?